src/Storages/ObjectStorage/DataLakes/Iceberg/IcebergMetadata.cpp

```diff
@@ -678,12 +678,12 @@ String composeList(
             if (field->has("direction"))
             {
                 auto d = field->getValue<String>("direction");
-                expr += (Poco::icompare(d, "desc") == 0) ? "DESC" : "ASC";
+                expr += (Poco::icompare(d, "desc") == 0) ? " DESC" : " ASC";
             }
             if (field->has("null-order"))
             {
                 auto n = field->getValue<String>("null-order");
-                expr += (Poco::icompare(n, "nulls-last") == 0) ? "NULLS LAST" : "NULLS FIRST";
```

Comment:
I would remove this part of the spec completely, it does not matter and does not have an equivalent in ClickHouse. Leave only ASC/DESC and that's it